CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Change history for ui-myprofile
 
-## [0.2.0](https://github.com/folio-org/ui-myprofile/tree/v0.2.0) (IN PROGRESS)
+## [0.2.0](https://github.com/folio-org/ui-myprofile/tree/v0.2.0) (2018-09-04)
```

Comment:
It's the 5th of September, not the 4th :-)

Reply:
Agree) That was in the shelf.